### PR TITLE
Libraries: Remove unused code

### PR DESCRIPTION
**Pull request by patrickelectric**
- Remove unused operations, variables and update some logics

### CODE REVIEW

File: libraries/AP_RangeFinder/AP_RangeFinder_Benewake.cpp [hunk 1]
```diff
@@ -123,16 +122,16 @@ bool AP_RangeFinder_Benewake::get_reading(uint16_t &reading_cm, bool &signal_ok)
                         } else {
                             // TF02 provides signal reliability (good = 7 or 8)
                             dist_reliable = (linebuf[6] >= 7);
-                        }
-                        if (dist_reliable) {
-                            sum_cm += dist;
-                            count++;
+
+                            if (dist_reliable) {
```
Comment (OXINARF) [inline libraries/AP_RangeFinder/AP_RangeFinder_Benewake.cpp]: This is wrong, TFmini doesn't add distance to sum_cm.

Comment (patrickelectric) [reply libraries/AP_RangeFinder/AP_RangeFinder_Benewake.cpp]: My bad, logic corrected.

**Comment by patrickelectric** [comment]
@OXINARF moving commits that will take more time to review in a second PR.
This one should be pretty straightforward now.

File: libraries/AP_Mount/AP_Mount_SToRM32_serial.cpp [hunk 2]
```diff
@@ -272,7 +272,6 @@ void AP_Mount_SToRM32_serial::parse_reply() {
         case ReplyType_ACK:
             crc = crc_calculate(&_buffer[1],
                                 sizeof(SToRM32_reply_ack_struct) - 3);
-            crc_ok = crc == _buffer.ack.crc;
```
Comment (OXINARF) [inline libraries/AP_Mount/AP_Mount_SToRM32_serial.cpp]: Can't we remove the whole thing? The `crc` gets unused now. It looks like it should be:

```
case ReplyType_ACK:
default:
    break;
```

**Comment by OXINARF** [comment]
Merged, thanks!

**Pull request merged**